<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTitleWidget</class>
 <widget class="QBaseWidget" name="QTitleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTitleWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTitleWidget{
	border-top-left-radius:8px;
	border-top-right-radius:8px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(58, 168, 155, 255), stop:1 rgba(129, 179, 77, 255));
}
QToolButton{
	border:none;
}
QToolButton#btnShare,QToolButton#btnFiles,QToolButton#btnUser,QLabel#lblTitle,QLabel#lblUserName{
	font-family: &quot;微软雅黑&quot;;
	font-size : 16px;
	color:white;
}
QToolButton#btnShare,QToolButton#btnFiles{
	padding-left:20px;
}
QToolButton#btnShare:hover,QToolButton#btnFiles:hover{
	background-color: rgba(255, 255, 255, 50);
}
/*
QToolButton#btnClose:hover{
	background-color: rgba(255,0,0,50);
}
*/
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblIcon">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../microlib.qrc">:/images/microlib</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTitle">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>微库</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QMaskLabel" name="lblUser">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblUserName">
       <property name="text">
        <string>用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnUser">
       <property name="text">
        <string>用户</string>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::DownArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>73</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnShare">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>72</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>公共目录</string>
       </property>
       <property name="icon">
        <iconset resource="../microlib.qrc">
         <normaloff>:/images/dir</normaloff>:/images/dir</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnFiles">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>72</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>我的文件</string>
       </property>
       <property name="icon">
        <iconset resource="../microlib.qrc">
         <normaloff>:/images/file</normaloff>:/images/file</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>72</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QToolButton" name="btnSet">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../microlib.qrc">
           <normaloff>:/images/setting</normaloff>:/images/setting</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QToolButton" name="btnMin">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../microlib.qrc">
           <normaloff>:/images/min</normaloff>:/images/min</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QToolButton" name="btnClose">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../microlib.qrc">
           <normaloff>:/images/close</normaloff>:/images/close</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btnDang">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../microlib.qrc">
           <normaloff>:/images/dang</normaloff>:/images/dang</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QBaseWidget</class>
   <extends>QWidget</extends>
   <header>qbasewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QMaskLabel</class>
   <extends>QLabel</extends>
   <header>Base/qmasklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../microlib.qrc"/>
 </resources>
 <connections/>
</ui>
